controller/src/enhancements/player.rs

```diff
@@ -422,264 +430,286 @@ impl Enhancement for PlayerESP {
     fn render(&self, settings: &AppSettings, ui: &imgui::Ui, view: &ViewController) {
         let draw = ui.get_window_draw_list();
         for entry in self.players.iter() {
-            let esp_settings = match self.resolve_esp_player_config(settings, entry) {
-                Some(settings) => settings,
-                None => continue,
-            };
-
-            let player_rel_health = (entry.player_health as f32 / 100.0).clamp(0.0, 1.0);
-            let player_2d_box = entry.calculate_player_box(view);
-
-            if esp_settings.skeleton {
-                let bones = entry.model.bones.iter().zip(entry.bone_states.iter());
-
-                for (bone, state) in bones {
-                    if (bone.flags & BoneFlags::FlagHitbox as u32) == 0 {
+            if let Some(local_pos) = self.local_pos {
```

Comment:
I would rather not cover the whole ESP in  
`if let Some(local_pos) = self.local_pos {`  
as if the local player pos is `None` this would disable everything.  
Instead you could do something like:  
```rust
let distance = if let Some(local_pos) = self.local_pos {
  ...
} else {
  0.0
};

 if esp_settings.near_players {
                    if distance > esp_settings.near_players_distance {
        continue;
    }
}

...
```

Reply:
done